### PR TITLE
Change version from 3.0-rc.2 to 3.0

### PR DESCRIPTION
**Pull request by andreas-junghanns**

### CODE REVIEW

**Comment by andreas-junghanns** [comment]
All: If we want to publish 3.0 today, we should get this PR approved by more than my eyes... please try to make the time soon today.

**Review by chrbertsch — COMMENTED** [review]
Are the two "dots" in the change in the xsd files  correct?  <xs:pattern value="3[.](.*)"/>
(I am no expert in XML)

File: schema/fmi3BuildDescription.xsd [hunk 1]
```diff
@@ -108,7 +108,7 @@ ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
             <xs:attribute name="fmiVersion" use="required">
                 <xs:simpleType>
                     <xs:restriction base="xs:normalizedString">
-                        <xs:pattern value="3[.](0-.*)"/>
+                        <xs:pattern value="3[.](.*)"/>
```
Comment (pmai) [inline schema/fmi3BuildDescription.xsd:111]:
```suggestion
                        <xs:pattern value="3[.](0|[1-9][0-9]*)([.](0|[1-9][0-9]*))?(-.+)?"/>
```

Comment (pmai) [reply schema/fmi3BuildDescription.xsd:111]: @andreas-junghanns This should cover the parts of semantic versioning we care about, while still allowing just "3.0", without any build-variants, which are not applicable... Still have to test this somewhat, though...

**Comment by andreas-junghanns** [comment]
Also added to fmi3TerminalsAndIcons.xsd.

File: schema/fmi3BuildDescription.xsd (hunk 1 above)
Comment (t-sommer) [reply schema/fmi3BuildDescription.xsd]: Shouldn't e.g. `3.0.1-beta.1` also be a valid version?

Comment (pmai) [reply schema/fmi3BuildDescription.xsd]: It is, or do I miss something?

**Comment by andreas-junghanns** [comment]
> Shouldn't the modeDescription schema for fmiVersion also be changed accordingly?

Please check the proposal...

File: schema/fmi3BuildDescription.xsd (hunk 1 above)
Comment (andreas-junghanns) [reply schema/fmi3BuildDescription.xsd]: > It is, or do I miss something?

We also allow now `3.0-gamma42Beta` - right?

**Review by chrbertsch — APPROVED** [review]

**Comment by t-sommer** [comment]
I ran a quick test with FMPy.
The following versions match the `fmiVersion` regex: `3.0`, `3.1`, `3.0.1`, `3.1-alpha.2`, `3.0.1-rc.1`
These don't: `3.x.1-rc.1`, `31`

**Review by t-sommer — APPROVED** [review]

**Pull request merged**